01week/datatypes.js

```diff
@@ -0,0 +1,102 @@
+//Write a JavaScript program to display the current day and time.
+function findTime(){
+const now = new Date(); //find date
+const date = (now.getMonth() + 1) + "/" + now.getDate(); //display month/day
+const time = now.getHours() + ":" + now.getMinutes(); //display hour:minute
+const dayArray = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]; //array for day of the week
+const day = dayArray[now.getDay()]; //get string day of week
+const dateTime = "It is " + day + ", " + date + " at " + time + "."; //display time and date in string
+return dateTime;
+}
+
+findTime();
+
+//Write a JavaScript program to convert a number to a string.
+//Create a funtion to return
+function numToString(num1,num2){
+  return ('The winner is number') + ' ' +(num1 + num2);
+}
+console.log(numToString(4,5));
+
+//Write a JavaScript program to convert a string to the number.
+//Function to return string of numbers
+function stringToNum(){
+  return (10+5+6);
+}
+console.log(stringToNum());
+
+//Write a JavaScript program that takes in different datatypes and prints out whether they are a:
+//Boolean
+function getUpdate(){
+  return true; //function return true statement
+}
+console.log(getUpdate());
+//Null
+function findUpdate(){
+ return null;  // Create a function with
+}
+console.log(findUpdate());
+
+//Undefined = Create a function to return a variable with an unassigned value
+function  add(){
+  return ''; //create function return without a value
+}
+console.log(add());
+
+//Number = Create a funtion to return a number.
+function addNums(num){
+  return num; //funtion return an interger
+}
+console.log(addNums(2));
+//NaN
+function data(){
+  return 1+1;
+}
+console.log (data());
+
+//String
+function profileString(name,city){
+  return (`My name is ${name}. I live in ${city}`);//return with String Interpolation (wrap variable with ${}
+}
+console.log(profileString('Sunny','Austin'));
+const name = 'Sunny';
+const city = 'Austin';
+console.log(name + ' ' + city);
```

Comment:
For all of your data type function you are hard coding what the data type is . For example simply returning 'true' is not checking that data type. Look up a method that with check the data type no matter what parameter I pass through. I should be able to pass through a number, string, boolean, ect...to one function and the same method can check them all.